plugins/inputs/logparser/grok/grok.go

```diff
@@ -285,6 +292,16 @@ func (p *Parser) ParseLine(line string) (telegraf.Metric, error) {
 			} else {
 				timestamp = time.Unix(0, iv)
 			}
+		case SYSLOG_TIMESTAMP:
+			ts, err := time.ParseInLocation("Jan 02 15:04:05", v, p.loc)
```

Comment:
Hi @danielnelson 
Should we use "Jan 2 15:04:05" instead of "Jan 02 15:04:05" ?

Reply:
I think maybe it should be `Jan _2 15:04:05`, but I can't seem to create a testcase that fails.  Do you have a example that is failing?

Reply:
If we try to parse "Sep  2 09:01:55 value=42" in test function TestSyslogTimestampParser(),
the test will show the following:
`=== RUN   TestSyslogTimestampParser
2018/06/22 09:36:41 E! Error parsing Sep 2 09:01:55 to time layout [SYSLOG_TIMESTAMP]: parsing time "Sep 2 09:01:55" as "Jan 02 15:04:05": cannot parse "2 09:01:55" as "02"`

Does this the example count?
Thanks.

Reply:
Ahh, the test was "passing" even though the parsing failed.  Here is the fix: https://github.com/influxdata/telegraf/pull/4334

Reply:
That fix looks nice. Thank you.